src/test/java/teammates/test/cases/webapi/GetSessionResultsAsCsvActionTest.java

```diff
@@ -0,0 +1,409 @@
+package teammates.test.cases.webapi;
+
+import org.apache.http.HttpStatus;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.DataBundle;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.exception.EntityNotFoundException;
+import teammates.common.exception.NullHttpParameterException;
+import teammates.common.exception.RequestExceedingRangeException;
+import teammates.common.util.Const;
+import teammates.test.driver.CsvChecker;
+import teammates.ui.webapi.action.CsvResult;
+import teammates.ui.webapi.action.GetSessionResultsAsCsvAction;
+
+/**
+ * SUT: {@link GetSessionResultsAsCsvAction}.
+ */
+public class GetSessionResultsAsCsvActionTest extends BaseActionTest<GetSessionResultsAsCsvAction> {
+
+    private DataBundle dataBundle;
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.RESULT_CSV;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return GET;
+    }
+
+    private void loadCustomBundle() {
```

Comment:
I see what you are doing here and it works! Although its a little different from how our other tests work.. maybe @xpdavid can help improve this.

Reply:
I think it is OK to create a new data file for this test case ad download as CSV is a big feature. One thing to note here is that please override the corresponding method to avoid `typicalDataBundle` from loading.

Reply:
The way I did this was only using the custom data bundle for session with statistics. By doing that I don't have to duplicate the typical data bundle for those test cases that do not require the custom data bundle. And that's why `typicalDataBundle` is still being loaded here.

Reply:
So it your `json` file is incremental based on `typicalDataBundle`?

Reply:
It only contains one session but with a MCQ question instead of a text question, so that there will be statistics that can be downloaded. I did not include those sessions that were originally in the `typicalDataBundle`. But this `json` file is only used for two test cases: `Typical case: results with statistics hidden` and `Typical case: results with statistics shown`. All other test cases use `typicalDataBundle` instead.